docs/developing.md

```diff
@@ -9,46 +9,94 @@ $ cd $(go env GOPATH)/src/github.com/deislabs/duffle
 ```
 
 ### Prerequisites
+
 You need:
-* A working Go 1.11.4 (or later) environment
```

Comment:
If I run `make build-all-bins` without `go` in the path, the command fails with:
```
bash -c "LDFLAGS=\"-w -s -X github.com/deislabs/duffle/pkg/version.Version=c6a43ae\" scripts/build.sh"
building linux-amd64
scripts/build.sh: line 11: go: command not found
make: *** [build-all-bins] Error 127
```
So it seems that go is still needed.

Reply:
You may have caught a mistake here. Will look.